src/main/java/com/interfaceict/ptrakcer/services/ProjectService.java

```diff
@@ -0,0 +1,23 @@
+//
+// Created by htrap19 on 1/8/22
+//
+
+package com.interfaceict.ptrakcer.services;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.data.jpa.repository.JpaRepository;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class ProjectService<T, Repo extends JpaRepository<T, Long>>
```

Comment:
Some duplication here, `Repo extends JpaRepository<T , Long>` also `ProjectEntityRepo` do the same functionality. It's better to use one of them.